src/cmd/services/m3dbnode/config/bootstrap.go

```diff
@@ -43,16 +44,87 @@ import (
 var (
 	// defaultNumProcessorsPerCPU is the default number of processors per CPU.
 	defaultNumProcessorsPerCPU = 0.125
+
+	// default order in which bootstrappers are run
+	// (run in ascending order of precedence).
+	defaultOrderedBootstrappers = []string{
+		// Filesystem bootstrapping must be first.
+		bfs.FileSystemBootstrapperName,
+		// Peers and commitlog must come before the uninitialized topology bootrapping.
+		commitlog.CommitLogBootstrapperName,
+		peers.PeersBootstrapperName,
+		uninitialized.UninitializedTopologyBootstrapperName,
+	}
+
+	// bootstrapper order where peers is prefered over commitlog.
+	preferPeersOrderedBootstrappers = []string{
```

Comment:
can we add `peerOnlyOrderedBootstrappers` or smth. that would not include commit logs?

Reply:
Added `ExcludeCommitLogBootstrapMode`